safekit-cluster-farm/README.md

```diff
@@ -0,0 +1,60 @@
+# Evidian SafeKit - Load Balancing Cluster with Failover in Azure - Farm Module
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fsafekit-cluster-farm%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+`Tags: load balancing, cluster, failover, high availability, business continuity, disaster recovery, evidian, safekit, farm`
+
+*   [Description](#description)
+*   [Deployed resources](#resources)
+*   [How to use](#use)
+*   [More information](#more)
+
+## <a name="description"></a>Description
+
+![How the Evidian SafeKit farm cluster implements load balancing and failover in Azure?](images/farmarch.png)
+
+On the previous figure,
+
+*   the critical application is running in all servers of the farm
+*   users are connected to a virtual IP address which is configured in the Azure load balancer
+*   SafeKit brings a generic health probe for the load balancer When the farm module is stopped in a server, the health probe returns NOK to the load balancer which stops the load balancing of requests to the server. The same behavior happens when there is a **hardware failure**
```

Comment:
fix punctuation

Reply:
still inconsistent here... end of bullet needs no punctuation but if anything should be a period - intra bullet make sure is correct